### PR TITLE
write sql schema from diagram.

### PR DESCRIPTION
**Pull request by Reggeditt**
- Created schema_based_on_diagram.sql file.
- added the FK indexes.
- Added a Join table to handle many-to-many relationship.

### CODE REVIEW

**Review by rotimiazeez — COMMENTED** [review]
# Hi @Reggeditt & @lorenzoworx, 

### While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project does not have a professional README file. Kindly ensure your key features are listed as required 🙏🏽. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

**Comment by binhussen** [comment]
@Reggeditt and @lorenzoworx customize the readme
Screenshot 2023-07-19 125408

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,44 @@
+/* Database schema to keep the structure of entire database. */
+
```
Comment (Ismailco) [inline schema_based_on_diagram.sql:2]: - [ ] Kindly consider creating indexes for all the FOREIGN KEYS in the schema, like 👉🏼 `CREATE INDEX ON medical_histories (patient_id);......`

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @Reggeditt, @lorenzoworx,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (Reggeditt) [reply schema_based_on_diagram.sql:2]: thanks for pointing that out. We'll get to work on it!

**Review by KDlamini — APPROVED** [review]
### Hi team @Reggeditt, @lorenzoworx 🙋‍♀️,  

Excellent work! 💯 
You have done a great job implementing all the required changes. 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!🎊 🥳 

## Highlights 🌞 

✔️  Professional pull request and README
✔️  The new tables are created in the `schema_based_on_diagram.sql` file
✔️  A "join table" for the many-to-many relationships is created
✔️  All FOREIGN KEYS are set and have indexes created

# Status: Approved ✔️ 

simon

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**